### PR TITLE
fix: confirm page title and subtitle size, image position and text color

### LINKED ISSUE #126: Improve Confirmation View
Explanation:

 The size of the elements in general is different from what you see in the design, place the notebook image on the site, check the color of the text used in the title confirm receipt, the font size of the text "we are done" and the subtitle together.

Screenshots:
view_start-desktop
image

 Checklist:
  
 - [ ] Locate notebook view.
 - [ ] Check and change the color of the text used in Confirm receipt.
 - [ ] Font size "we are done" and subtitle.
 - [ ] Verify screen sizes:
 
### References:

How to get to this view in the application: /confirm/1

https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=781%3A696

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

Increased width image notebook, updated title and subtitle color, size, and weight.

fixes #126 

#### Screenshots

image (1)

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### How to test it

Go to 'confirm/1'

### Checklist:

- [x] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [x] Doesn't have unnecessary comments or debugging code.
- [x] The branch is updated with the last dev branch changes.

### CODE REVIEW

File: src/components/Text.jsx [hunk 1]
```diff
@@ -133,13 +133,22 @@ const Text =
...
+      `}
+      
   ${
   /** @param {TextProps} props */
   props =>
       props.bold &&
       css`
         ${text['600']};
       `}
+
```
Comment (Israel-Laguan) [inline src/components/Text.jsx]: extra line

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]
Good work

**Review by e-azocar — CHANGES_REQUESTED** [review]
On some screen sizes this happens. For example:
width1345px 
height 600px
image

I think is problem with height, because this doesn't happen on higher. For example:
width1345px 
height 800px
image

**Comment by Eghinner** [comment]
> On some screen sizes this happens. For example: width1345px height 600px image
> 
> I think is problem with height, because this doesn't happen on higher. For example: width1345px height 800px image
heigth

**Review by e-azocar — APPROVED** [review]

**Pull request merged**